pkg/mons/mon-endpoints.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2023 kubectl-rook-ceph
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package mons
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kubectl-rook-ceph/pkg"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const monConfigMap = "rook-ceph-mon-endpoints"
+
+func GetMonEndpoint(clusterNamespace string) {
+	// Get Kubernetes Client
+	_, client := pkg.GetKubeClient()
+
+	monCm, err := client.ConfigMaps(clusterNamespace).Get(context.TODO(), monConfigMap, v1.GetOptions{})
+	if err != nil {
+		log.Errorf("failed to get mon configmap %s %v", monConfigMap, err)
+	}
+
+	monData := monCm.Data["data"]
+	fmt.Println(monData)
```

Comment:
To match the current output of the mons command, do we need to do any parsing of the endpoints?

Reply:
no we are getting the output as expected `a=10.96.52.53:6789`

Reply:
Looks like the mon name is stripped out: 
```
$ kubectl rook-ceph mons
10.97.4.37:6789,10.108.120.61:6789,10.100.156.174:6789
```

Reply:
Correct, it was in the current shell script. We just need to maintain compatibility with the script output for the same commands. Perhaps in the future we could add an option that would keep the mon names in the output.

Reply:
done